app/experimenter/experiments/migrations/0153_auto_20210127_1855.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 3.1.5 on 2021-01-27 18:55
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("experiments", "0152_auto_20210119_1856"),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
Since we're altering these fields and the valid choices for them, do we need a manual migration to convert any existing data in the DB?

Reply:
Yeah I was checking on that, staging doesn't matter, and there's only a single fenix experiment on prod with a channel so we can just change it in the admin.

Reply:
This also throws an error in GQL queries:

```
    errors	[ {…} ]
    0	Object { message: "Expected a value of type \"NimbusExperimentChannel\" but received: org.mozilla.firefox.beta", path: […] }
    message	"Expected a value of type \"NimbusExperimentChannel\" but received: org.mozilla.firefox.beta"
    path	[ "experimentBySlug", "channel" ]
    0	"experimentBySlug"
    1	"channel"
```

Reply:
Ooohhh iiiiiiinteresting.....  Apollo is throwing that?

Reply:
Then we do have to do a data migration, good catch 👍

Reply:
Yeah local just `make refresh`, prod only has a single one, but stage I didn't know it would break gql like that, so yeah we'll have to do a little db migrate.

Reply:
> Ooohhh iiiiiiinteresting..... Apollo is throwing that?

Not Apollo, that's in the HTTP response from the django-graphene side of things

Reply:
Oh wow!  Interesting.